spec/requests/api/graphql/mutations/create_consignment_offer_response_spec.rb

```diff
@@ -0,0 +1,191 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/gravity_helper'
+
+describe 'createConsignmentOfferResponse mutation' do
+  let!(:offer) { Fabricate :offer }
+
+  let(:token) do
+    user_id = offer.submission.user.gravity_user_id
+    payload = { aud: 'gravity', sub: user_id, roles: 'user' }
+    JWT.encode(payload, Convection.config.jwt_secret)
+  end
+
+  let(:headers) { { 'Authorization' => "Bearer #{token}" } }
+
+  let(:mutation_inputs) do
+    <<-INPUTS
+    {
+      offerId: "#{offer.id}",
+      intendedState: ACCEPTED
+    }
+    INPUTS
+  end
+
+  let(:mutation) do
+    <<-GRAPHQL
+    mutation {
+      createConsignmentOfferResponse(input: #{mutation_inputs}){
+        consignmentOfferResponse {
+          id
+        }
+      }
+    }
+    GRAPHQL
+  end
+
+  describe 'invalid requests' do
+    context 'with an unauthorized request' do
```

Comment:
[non-blocking] Another useful test here would be `context 'with someone else's offer'` .. `it 'returns an error for that request'`. It _looks_ like that's accounted for in the underlying mutation, but it'd be nice to have a test to prove it. 

I would merge without this test but if you have the time & feel like it's easy to add before merging, it's a welcome addition.